gr-digital/include/gnuradio/digital/constellation.h

```diff
@@ -169,7 +170,7 @@ class DIGITAL_API constellation : public std::enable_shared_from_this<constellat
      *        generating the LUT.
      */
     void set_soft_dec_lut(const std::vector<std::vector<float>>& soft_dec_lut,
```

Comment:
API change

Reply:
this one has a default argument no? I'm sorry, I'm new to the practices here.

Reply:
I don't see a default on line 172, where `npwr` was added to `set_soft_dec_lut()`.